test/jdk/javax/swing/AbstractButton/5049549/bug5049549.java

```diff
@@ -38,19 +38,31 @@
 import javax.swing.JPanel;
 import javax.swing.SwingUtilities;
 import javax.swing.UIManager;
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.Graphics;
+import java.awt.image.BufferedImage;
 
 public class bug5049549 {
 
-    private static ImageIcon DE = new ImageIcon(bug5049549.class.getResource("DE1.gif"));
```

Comment:
```java
    private static ImageIcon DE = generateImage("DE");
```
Preserve the original code style? I have no strong preference between the two in this case.